client/my-sites/upgrades/checkout-thank-you/failed-purchase.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import { localize } from 'i18n-calypso';
+import PurchaseDetail from 'components/purchase-detail';
+
+const FailedPurchase = ( { failedPurchases, translate } ) => {
+	const classes = {
+		'checkout-thank-you__header': true,
+	};
+
+	const description = (
```

Comment:
Good idea to extract `description`, but even better would be to put it into a separate method. This one is long enough as it is ;)